pkg/schemas/csm/xnames.go

```diff
@@ -0,0 +1,223 @@
+package csm
```

Comment:
The hms-xnames project is fairly complete. https://github.com/Cray-HPE/hms-xname
Did you consider using this?

Alternately what you have here is fine. It has the advantage of being simpler and more targeted.

Reply:
That's because I thought we would exclusively stick with our own schemas repo. The only structs in those references are

- `csm.XNameComponents`
- `csm.RedfishEndpoint`

Both of which probably have parallels in hms-xname. If that's the way we want to go, I don't anticipate that it would be too hard to do it in `ochami`.

Reply:
Does that mean, in the context of OpenCHAMI, should SMD "own" the xname schema?